### PR TITLE
MUI changes

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Add styling for responsive table in larger screen sizes.
2. Add helper classes ( `mui-list--aligned`, `mui--d-inlineblock`)
3. Remove the scroll lock class added by mui.js on opening select menu. Fixes the body scroll lock for select2
4. Minor css fixes for mui-form

### CODE REVIEW

File: baseframe/templates/baseframe/mui/forms.html.jinja2 [hunk 1]
```diff
@@ -125,7 +138,7 @@
       <p class="alert__text"><i class="material-icons alert__icon">error</i> {% trans %}Please review the indicated issues{% endtrans %}</p>
     </div>
   {%- endif %}
-  {% if message %}<p>{{ message }}</p>{% endif %}
+  {% if message %}<p{{ message }}</p>{% endif %}
```
Comment (jace) [inline baseframe/templates/baseframe/mui/forms.html.jinja2]: `<p>` tag is incomplete. Typo?

Comment (vidya-ram) [reply baseframe/templates/baseframe/mui/forms.html.jinja2]: Fixed

File: baseframe/static/sass/baseframe-material/_modal.scss [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+  .modal__body {
+    @extend .mui--text-body2;
+  }
+}
+
+.jquery-modal.blocker.current {
+  > .modal {
+    display: inline-block;
+  }
 }
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply baseframe/static/sass/baseframe-material/_modal.scss]: Fixed

File: baseframe/static/sass/baseframe-material/_layout.scss [hunk 3]
```diff
@@ -36,26 +29,77 @@ h1, h2, h3 {
...
-    background-color: $mui-overlay-bg-color;
-    z-index: 2;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background-color: $mui-overlay-bg-color;
+  z-index: 2;
+}
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply baseframe/static/sass/baseframe-material/_layout.scss]: Fixed

File: baseframe/static/js/baseframe-material.js [hunk 4]
```diff
@@ -4,6 +4,11 @@ function activate_widgets() {
   // Activate select2.js for non-mobile browsers
   if (!Modernizr.touch) {
     $('select:not(.notselect)').select2();
+
+    //Remove the scroll lock class added by mui.js on opening select menu
+    $('select:not(.notselect)').on("select2:select", function(e) {
```
[the remark this answers is not in the document]
Comment (vidya-ram) [reply baseframe/static/js/baseframe-material.js]: Filed an issue https://github.com/hasgeek/baseframe/issues/196

**Pull request merged**